index.html

```diff
@@ -39,12 +39,405 @@ <h1 class="name">
                         </p>
                         
                        </div>
-                       
-                            
+                </div>
+            </section>
+
+            <section class="page1">
+
+                     <div class="geometry1">
+                    <img src="./images/image_geometry_2.png" alt="geometry">
+                  </div>
+                     <h1 class="projects">
+                    Projects
```

Comment:
- [ ] Please make sure you fix the button that comes above the `Project` title. This is due to the position it has that makes it overflow its container. Check how it looks:

![image](https://user-images.githubusercontent.com/80612925/177206881-058a1fe1-8ecb-44ea-a832-3b24967ae165.png)